GramAddict/core/interaction.py

```diff
@@ -11,9 +11,9 @@
 
 logger = logging.getLogger(__name__)
 
-TEXTVIEW_OR_BUTTON_REGEX = "android.widget.TextView|android.widget.Button"
-FOLLOW_REGEX = "Follow|Follow Back"
-UNFOLLOW_REGEX = "Following|Requested"
+BUTTON_REGEX = "android.widget.Button"
+FOLLOW_REGEX = "^Follow|^Follow Back"
```

Comment:
Follow Back is unnecessary since Follow will already match it

Reply:
Fair - that was just how it was originally.

Reply:
I had a comment but is not critical. You can remove this in another future commit.